statistics/handle/unlock_stats_handler.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handle
+
+import (
+	"context"
+
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/parser/ast"
+	"github.com/pingcap/tidb/statistics/handle/cache"
+	"github.com/pingcap/tidb/util/sqlexec"
+	"go.uber.org/zap"
+)
+
+// RemoveLockedTables remove tables from table locked array
+func (h *Handle) RemoveLockedTables(tids []int64, pids []int64, tables []*ast.TableName, maxChunkSize int) (string, error) {
```

Comment:
I recommend removing `maxChunkSize` here. This operation of removing locked tables is very lightweight, so this argument has almost no impact on this operation. I recommend removing it to simplify our code, and we can just use an arbitrary number here.

Reply:
Sounds good to me.

Reply:
Addressed in [`e83859a` (#46478)](https://github.com/pingcap/tidb/pull/46478/commits/e83859aeb36a1bc0ed05e21adc55dd77eb41d79c)